examples/linux/pi-network.yml

```diff
@@ -0,0 +1,48 @@
+# Ensure nri-flex is installed in /var/db/newrelic-infra/newrelic-integrations/bin
+# Place pi-network.yaml in /etc/newrelic-infra/integrations.d and restart the newrelic-infra agent
+# Tested on an armv7l Rapsberry Pi
+---
+integrations:
+  - name: nri-flex
+    config:
+      name: interfaceLinkStatus
+      apis:
+        # Let's get a list of interfaces and save them for the next command
+        - name: interfaceLink
+          event_type: interfaceLookup
+          commands:
+            - run: ip link | egrep '^[0-9]' | awk '{gsub(/:/,"",$2); print $2}'
+              set_header: [interface]
+              split: horizontal
+              split_by: ":"
+          ignore_output: true
+        # Use the lookup to loop through the interface list and parse the output
+        - name: interfaceLink
+          event_type: interfaceLinkSample
```

Comment:
same as above; are both of these required?

Reply:
Updated.